File: CS/winforms-ai-extensions/RichEditForm.cs
using DevExpress.XtraBars.Ribbon;
using DevExpress.XtraEditors;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace WinForms_AI_Extensions
{
    public partial class RichEditForm : DevExpress.XtraBars.Ribbon.RibbonForm
    {
        public RichEditForm()
        {
            InitializeComponent();
            RibbonControl ribbon = richEditControl1.CreateRibbon();
            this.Controls.Add(ribbon);
            richEditControl1.LoadDocument("WindowsForms.docx");
        }
    }
}

File: VB/winforms-ai-extensions/RichEditForm.vb
Imports DevExpress.XtraBars.Ribbon
Imports System.ComponentModel
Imports System.Drawing
Imports System.Linq

Namespace WinForms_AI_Extensions

    Public Partial Class RichEditForm
        Inherits RibbonForm

        Public Sub New()
            InitializeComponent()
            Dim ribbon As RibbonControl = richEditControl1.CreateRibbon()
            Me.Controls.Add(ribbon)
            richEditControl1.LoadDocument("WindowsForms.docx")
        End Sub
    End Class
End Namespace
